src/assets/components/Movie.css

```diff
@@ -0,0 +1,432 @@
+html,
+body {
+  background-color: #000;
+}
+
+a:link,
+visited,
+hover,
+active {
+  text-decoration: none;
+}
+
+h1 {
+  text-align: center;
+  align-items: center;
+  color: white;
+  padding: 10px;
+  font-size: 80px;
+}
+
+
+img {
+  width: 100%;
+  border-radius: 15px;
+  height: auto;
+  cursor: pointer;
+  transition: transform 0.3s ease-in-out, filter 0.3s ease-in-out;
+}
+
+/****navbuttons ***/
+.paginate {
+  display: flex;
+  justify-content: center;
+  margin: 1vh 0;
+
+}
+
+.page-number {
+  display: flex;
+  justify-content: center;
+  color: white;
+  font-size: 1.5em;
+
+}
+
+.previous-btn {
+  background-image: url(../../../public/back-btn.svg);
+  filter: url(../../../public/back-btn.svg);
+  filter: none;
+  width: 32px;
+  height: 32px;
+  display: flex;
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin: 5vh 1vw;
+  filter: invert(100%);
+}
+
+.next-btn {
+  background-image: url(../../../public/back-btn.svg);
+  filter: url(../../../public/back-btn.svg);
+  filter: none;
+  width: 32px;
+  height: 32px;
+  display: flex;
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin: 5vh 1vw;
+  filter: invert(100%);
+  transform: rotate(180deg);
+}
+
+
+
+/***** main page *****/
+ul {
+  list-style-type: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: 20px;
+}
+
+li {
+  text-align: center;
+  z-index: 0;
+}
+
+li:hover img {
+  transform: scale(1.05);
+  filter: brightness(0.2);
+}
```

Comment:
I really like this hover animation! It's subtle but works really well with the rest of the styling